### PR TITLE
Init CI (Build and Tests)

### PR DESCRIPTION
**Pull request by kamilbaczek**

### CODE REVIEW

**Comment by kamilbaczek** [comment]
https://github.com/meaboutsoftware/SuperSimpleArchitecture/pull/7 when this PR will be merged tests will pass .

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    branches: [ "main" ]
+
+jobs: 
+  build:
+    defaults:
+      run:
+        working-directory: Src
+    runs-on: ubuntu-latest
+    name: Build and test
+    steps:
```
Comment (meaboutsoftware) [inline .github/workflows/ci.yml:16]: what about:

jobs:
build:
test:

? I mean to separate test from build, then you have first build, then test (you can use "needs: [build]")

**Review by meaboutsoftware — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (kamilbaczek) [reply .github/workflows/ci.yml:16]: Yes I thought about this. I planned to do it in next MR. I can do it in first version. No problem :D

Comment (kamilbaczek) [reply .github/workflows/ci.yml:16]: Done. 

I love how it looks with GitHub client :D I have never seen this window before :D 
image

**Review by meaboutsoftware — APPROVED** [review]

**Pull request merged**